### PR TITLE
only retrieves orders for current ledger

### LINKED ISSUE #100: Current purchase order not always returned for bib
The current logic of retrieving the latest order just looks for the order that was created most recently. However, sometimes the active order was created before a cancelled purchase order. Orders need to be filtered to those attached to the current ledger.

### PR DESCRIPTION
**Pull request by mzelesky**
Closes #100. In addition, the get_current_issues method returned an array of arrays; it now will only return an array of strings. Finally, if there are any current issues received, no order status is shown.

### CODE REVIEW

File: lib/voyager_helpers/liberator.rb [hunk 1]
```diff
@@ -586,22 +588,41 @@ def callno_from_852(field_852)
         call_no
       end
 
+      # @return [Fixnum] A ledger ID
+      def get_ledger(conn=nil)
+        query = VoyagerHelpers::Queries.ledger
+        date = Date.today
+        year = date.year
+        month = date.month
+        year += 1 if month > 6
```
Comment (tampakis) [inline lib/voyager_helpers/liberator.rb]: Is this related to fiscal year?

**Comment by tampakis** [comment]
Works great!

File: lib/voyager_helpers/liberator.rb (hunk 1 above)
Comment (mzelesky) [reply lib/voyager_helpers/liberator.rb]: Yes, it's the best way I could think of getting the current fiscal year.

Comment (mzelesky) [reply lib/voyager_helpers/liberator.rb]: Around the end of the fiscal year (June-August), 2 ledgers are open at the same time. So the correct one needs to be determined by some sort of date calculation.
